src/Compilers/Core/Portable/Compilation/CommonModuleCompilationState.cs

```diff
@@ -50,7 +53,7 @@ internal void SetStateMachineType(TMethodSymbol method, TNamedTypeSymbol stateMa
             }
         }
 
-        internal bool TryGetStateMachineType(TMethodSymbol method, out TNamedTypeSymbol stateMachineType)
+        internal bool TryGetStateMachineType(TMethodSymbol method, [NotNullWhen(returnValue: true)] out TNamedTypeSymbol? stateMachineType)
```

Comment:
📝 At this point I've stopped putting `returnValue:`. I'm not going back to change the existing ones but it's up to you on how to handle future files.

Reply:
Does omittting return value impact the behavior?

Reply:
No, it was just for readability while we were learning to use them.